src/it/java/uk/gov/justice/digital/service/operationaldatastore/OperationalDataStoreServiceIntegrationTest.java

```diff
@@ -113,14 +115,15 @@ void setUp() throws Exception {
         givenSchemaExists(inputSchemaName, testQueryConnection);
         givenTablesToWriteToOperationalDataStoreTableNameIsConfigured(jobArguments, configurationSchema + "." + configurationTable);
         givenTablesToWriteToOperationalDataStore(configurationSchema, configurationTable, inputSchemaName, inputTableName, testQueryConnection);
+        lenient().when(jobArguments.getOperationalDataStoreJdbcBatchSize()).thenReturn(OPERATIONAL_DATA_STORE_JDBC_BATCH_SIZE_DEFAULT);
```

Comment:
Is lenient necessary here or could it be replaced with normal stubbing?

Reply:
Most of the tests but not quite all require it - so i could move it out of setup and in to the individual tests if you think that is preferable?

Reply:
Not really needed, I just wanted to confirm you were aware of this being used.

Reply:
Approved.